rs/nns/governance/canister/governance.did

```diff
@@ -517,7 +517,7 @@ type ListNodeProvidersResponse = record {
   node_providers : vec NodeProvider;
 };
 
-type ListProposalInfo = record {
+type ListProposalInfoRequest = record {
```

Comment:
Why not keep the name (throughout)? This change (within governance.did) in particular is disruptive to ppl (e.g. NNS Dapp) who generate code from our .did file(s).

Anyway, I don't feel that strongly one way or the other. Do as you see fit.